lib/tutorials/en_US/logging.md

```diff
@@ -1,46 +1,88 @@
+You want to call `request.log()` whenever in the context of a request, such as a route handler, request lifecycle extension, or authentication scheme. The method takes two argument:
 
-`tags` is a string or array of strings used to briefly identify the event. Think of them like log levels, but far more expressive. For example, you could tag an error retrieving data from your database like the following:
+`tags`: a string or an array of strings (e.g. `['error', 'database', 'read']`) used to identify the event. Tags are used instead of log levels and provide a much more expressive mechanism for describing and filtering events.
+
+`data`: (optional) an message string or object with the application data being logged. If data is a function, the function signature is `function()` and it called once to generate (return value) the actual data emitted to the listeners.
+
+For example:
+
+```js
+server.route({
+    method: 'GET',
+    path: '/',
+    handler: function (request, h) {
 
-```javascript
-server.log(['error', 'database', 'read']);
+        request.log('error', 'Event error');
+        return 'Hello World';
+    }
+});
 ```
+In this example, if there is a request-specific event with a tag of `error`, the event will get logged. We also send a `data` parameter of `Event error`. This can be anything you want, such as an error message or any other details.  
 
-Any log events that hapi generates internally will always have the `hapi` tag associated with them.
+### <a name="server.log" /> server.log()
 
-The second parameter, `data`, is an optional string or object to log with the event. This is where you would pass in things like an error message, or any other details that you wish to go along with your tags.
+`server.log()` is used when you have no specific request in scope, for instance, just after your server has started or inside a plugin's `register()` method.
 
-Additionally `server.log()` accepts a third `timestamp` parameter. This defaults to `Date.now()`, and should only be passed in if you need to override the default for some reason.
+`server.log()` takes three parameters, `(tags, data, timestamp)`. The `tags` and `data` parameters are exactly the same as in `request.logs()`. The `timestamp` parameter defaults to `Date.now()` and should only be passed in if you need to override the default for some reason.  
 
-### Retrieving and displaying logs
+```js
+const Hapi = require('hapi');
+const server = Hapi.server({ port: 80 });
 
-The hapi server object emits events for each log event. You can use the standard EventEmitter API to listen for such events and display them however you wish.
+server.events.on('log', (event, tags) => {
+
+    if (tags.error) {
+        console.log(event);
+    }
+});
+
+server.log(['test', 'error'], 'Test event');
+```
+In this example, we see that server will log any event with `tags` `'test'` and `'error'`. Since there is no specific request in scope in this example, we use `server.log()`.
```

Comment:
```suggestion
In this example, we see that server will log an event with `tags` `'test'` and `'error'`. Since there is no specific request in scope in this example, we use `server.log()`.
```